package.json

```diff
@@ -80,10 +80,10 @@
     "tap-bark": "^1.0.0",
     "ts-node": "^6.2.0",
     "tslint": "^5.11.0",
-    "typedoc": "^0.11.1"
+    "typedoc": "^0.11.1",
+    "@types/node": "^12.7.8",
```

Comment:
`"@types/node": "^12.7.8",` -> `@types/node": "^12.7.8"`
Using a trailing comma at the end of a JSON object is illegal 😜